src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterfaceSecurity.cs

```diff
@@ -112,7 +119,50 @@ class ConsoleHostUserInterface : System.Management.Automation.Host.PSHostUserInt
             {
                 return null;
             }
+            if (reEnterPassword)
+            {
+                reEnterPasswordPrompt = StringUtil.Format(ConsoleHostUserInterfaceSecurityResources.PromptForCredential_ReEnterPassword, userName);
+
+                //
+                // now, prompt to re-enter the password
+                //
```

Comment:
Please use one line comments like:
```suggestion
                // Now, prompt to re-enter the password.
```